src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
```

Comment:
class ColorSupplier have not any state(fields) and class behavior(non statick methods), also there are no logical inheritance between Lottery and ColorSupplier in my mind, so inheritance is not needed here